lib/src/config/themes/light_theme.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+ThemeData lightTheme() {
```

Comment:
Maybe you can create and structure like a Json file that contains all the themes. like:
```
{
   "themes" : [
         {
               "themeName": "light",
               "canvasColor": "...",
               "colorScheme": {
                    "primary": "...",
                    "secondary": "...",
               }
         }
   ]
}
```

Reply:
And also you can define on that json the dark theme, or any new theme that you want to use